test/integration/targets/ec2_classic/tasks/ec2_group.yml

```diff
@@ -0,0 +1,74 @@
+- block:
```

Comment:
I wonder if these will be easily overlooked/forgotten here since they don't correspond to a module. Would it be better to put these as a new task file in the ec2_group target, and add a `when` to conditionally include them depending on the account attribute?

Reply:
Ah yeah, good point.